In-house patch to remove gzipping of man pages

The patch will not be sent to upstream.

--- rabbitmq-server-3.7.8/Makefile	2018-11-01 07:20:24.343716610 +0000
+++ rabbitmq-server-3.7.8/Makefile	2018-11-01 07:19:53.312143546 +0000
@@ -347,8 +347,8 @@ install-man: manpages
 	for section in $$sections; do \
 		mkdir -p $(DESTDIR)$(MANDIR)/man$$section; \
 		for manpage in $(DEPS_DIR)/rabbit/docs/*.$$section; do \
-			gzip < $$manpage \
-			 > $(DESTDIR)$(MANDIR)/man$$section/$$(basename $$manpage).gz; \
+			cp $$manpage \
+			 $(DESTDIR)$(MANDIR)/man$$section/$$(basename $$manpage); \
 		done; \
 	done
 
